### PR TITLE
doc: update README about start server with example graph

### PR DESCRIPTION
**Pull request by aroundabout**
## Purpose of the PR

- update readme about start `server` with example graph

## Main Changes

- update readme about start `server` with example graph
- update the link of the subproject of `toolchain`

## Verifying these changes

- [x] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [x]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [x]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +57,7 @@ The project [doc page](https://hugegraph.apache.org/docs/) contains more informa
 and provides detailed documentation for users. (Structure / Usage / API / Configs...)
 
 And here are links of other **HugeGraph** component/repositories:
-1. [hugegraph-toolchain](https://github.com/apache/incubator-hugegraph-toolchain) (graph **loader/dashboard/tool/client**)
+1. [hugegraph-toolchain](https://github.com/apache/incubator-hugegraph-toolchain) (graph **[loader](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-client)/[dashboard](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-hubble)/[tool](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-tools)/[client](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-client)**)
```
Comment (imbajin) [inline README.md:60]: use relative path (refer README in root)

**Review by imbajin — COMMENTED** [review]
Also update the instrinstruction in root (enhance the desc)

image

**Review by javeme — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (liuxiaocs7) [inline README.md:60]: loader link seems not right?

**Review by liuxiaocs7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aroundabout) [inline README.md:60]:
```suggestion
1. [hugegraph-toolchain](https://github.com/apache/incubator-hugegraph-toolchain) (graph **[loader](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-loader)/[dashboard](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-hubble)/[tool](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-tools)/[client](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-client)**)
```

**Review by imbajin — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Pull request merged**